### DISCUSSION TITLE
renaming folders

### DISCUSSION THREAD

**Discussion by denppa** [beetbox/beets#4041] [Q&A] [answered]
So sometimes my folders containing the album might not be the way  I want it.

I want it to be smth like this:

`$artist - $album_name - $year`

And I want the separators to be the `-`

Which plugins do I need to install?

**Comment by denppa** [comment, marked as answer]
https://beets.readthedocs.io/en/stable/guides/advanced.html?highlight=folder#customize-your-file-and-folder-names

**Reply by jef** [reply]
Looks like you answered your own question? :grinning: Good luck!
